packages/server/src/api/storage/sql/sql.js

```diff
@@ -79,6 +79,7 @@ function createSequelize(options) {
   const sequelizeOptions = {
     operatorsAliases: false,
     logging: () => {},
+    type: Sequelize.QueryTypes.BULKUPDATE,
```

Comment:
what does this do? I can't find `type` in the constructor options https://sequelize.org/master/class/lib/sequelize.js~Sequelize.html#instance-constructor-constructor

Reply:
This is an option for the queries that will made with sequelize: [QueryTypes](https://sequelize.org/v4/variable/index.html#static-variable-QueryTypes)

Without the option it's not possible to run all the migrations with "umzug" because there you made a bulk update. 

But i read the code, documentation again and I found out that it's better to set the "queryType" to the bulk options in the migration scripts itself. 

The type will dropped out of the sql.js file and added to the bulk operations direct